utils/objects/Labels.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+import numpy as np
+
+
+@dataclass
+class RoiLabel:
+    label: str
+    index: int
+    color: str
+    RGB: list
```

Comment:
The computation you have for populating this in `Creators`:
```python
 RGB=[int("fbbc05"[i: i + 2], 16) for i in [1, 3, 5]],
```
Is pretty involved, maybe it should go in here and have a bit of an explanation as well